erpnext/patches.txt

```diff
@@ -708,4 +708,5 @@ erpnext.patches.v13_0.move_doctype_reports_and_notification_from_hr_to_payroll #
 erpnext.patches.v13_0.move_payroll_setting_separately_from_hr_settings #22-06-2020
 erpnext.patches.v13_0.check_is_income_tax_component #22-06-2020
 erpnext.patches.v12_0.add_taxjar_integration_field
+erpnext.patches.v12_0.delete_report_requested_items_to_order
```

Comment:
add this patch in v13 folder for patches. New reports are only for develop branch (v13)